front/app/modules/commercial/insights/admin/containers/Insights/Edit/InputsTable/EmptyState.tsx

```diff
@@ -22,19 +23,35 @@ const StyledEmptyState = styled.div`
   font-weight: bold;
   line-height: 25px;
   padding-top: 80px;
+  text-align: center;
   svg {
     margin-bottom: 20px;
     height: 35px;
   }
+  p {
+    font-weight: normal;
+    font-size: ${fontSizes.small}px;
+    margin-top: 16px;
+  }
 `;
 
-const EmptyState = ({ intl: { formatMessage } }: InjectedIntlProps) => {
+const EmptyState = ({
+  intl: { formatMessage },
+  location: { query },
+}: InjectedIntlProps & WithRouterProps) => {
   return (
     <StyledEmptyState data-testid="insightsInputsTableEmptyState">
       <Icon name="blankPage" />
-      {formatMessage(messages.inputsTableEmpty)}
+      {query.category ? (
+        <>
+          {formatMessage(messages.inputsTableCategoryTitle)}
```

Comment:
Maybe that title should be inside some H element ? Admin side is not up to any accessibility standards anyways so not blocking

Reply:
Absolutely, very good point!